COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2015 The NixOS and nixpkgs authors
```

Comment:
The Nix code lives in a different repo, https://github.com/NixOS/nix, so no need to add it here.